generator/tree/nodes/trivial/constant.py

```diff
@@ -1,11 +1,14 @@
 from generator.tree.nodes.node import Node
 from generator.tree.helpers.basictype import BasicType
-
+from generator.tree.errors import InvalidConstantValueError
 
 class Constant(Node):
     def __init__(self, name, properties=None, own_schema=None):
```

Comment:
The default value of `None` no longer works for `properties` now since you access the `value` and `type` members.

Reply:
Done